index.js

```diff
@@ -13,6 +14,7 @@ module.exports = (target, opts) => {
 	let appArgs = [];
 	let args = [];
 	const cpOpts = {};
+	const isWSL = (os.type() === 'Linux' && os.release().indexOf('Microsoft') > -1);
```

Comment:
Yup, I did:

![2017-04-16_14-28-07](https://cloud.githubusercontent.com/assets/1459603/25074504/05931ffe-22b1-11e7-8d63-49ae17e6fe8c.png)

Are you on the latest version of Windows 10 and Ubuntu distro for WSL? It might be possible that the Anniversary update WSL, which uses Ubuntu 14.04, might not include it. Updating to the Creator's Update didn't automatically upgrade the 14.04 to 16.04 either, so that could also be an issue. In any case, I'm fine with using your module if you'd prefer.